interface/billing/sl_eob_search.php

```diff
@@ -166,10 +166,16 @@ function upload_file_to_client($file_to_send) {
   // sleep one second to ensure there's no follow-on.
   sleep(1);
 }
-function upload_file_to_client_pdf($file_to_send) {
+function upload_file_to_client_pdf($file_to_send, $aPatFirstName = '', $aPatID = null, $flagCFN = false) { //modified for statement title name
   //Function reads a HTML file and converts to pdf.
 
-  global $STMT_TEMP_FILE_PDF;
+   $aPatFName = preg_replace("/[^A-Za-z]/", '', $aPatFirstName); //modified for statement title name
```

Comment:
Recommend using the following function for this instead:
convert_safe_file_dir_name()
(this function can be used anywhere)
https://github.com/openemr/openemr/blob/master/library/sanitize.inc.php#L32